crates/miden-lib/src/account/interface/mod.rs

```diff
@@ -289,6 +289,27 @@ impl From<&Account> for AccountInterface {
     }
 }
 
+/// Helper function to extract public keys from an account
+pub fn get_public_keys_from_account(account: &Account) -> Vec<Word> {
```

Comment:
I would not add such free-standing functions to the public `miden-lib` API. I think we should either make it an inherent method on `AccountInterface` or move it behind `#[cfg(test)]` or `testing`.

It seems to be only used for testing purposes in miden-lib and miden-testing, so I would move it to `crates/miden-lib/src/testing/account_interface.rs`.